### ISSUE TITLE
Can I get a single character box？

### ISSUE THREAD

**Issue by HHeracles** [MhLiao/MaskTextSpotterV3#24]
Hello @MhLiao ，We  can get each character's mask in your program，but each character's mask only covers a part of the character. Now, I want to get the feature map for each character area, so I need to get the boxes for each character.  First，I want to save char boxes. Can we get character boxes without changing the model structure? If so, how to do?

**Comment by MhLiao** [comment]
You can get the char boxes from `char_mask` like [this](https://github.com/MhLiao/MaskTextSpotterV3/blob/9bd595e40c693ced0dc8ce4f33d65048424189e5/maskrcnn_benchmark/engine/text_inference.py#L391). The `char_mask` can obtain like [this](https://github.com/MhLiao/MaskTextSpotterV3/blob/9bd595e40c693ced0dc8ce4f33d65048424189e5/maskrcnn_benchmark/engine/text_inference.py#L86).

**Comment by HHeracles** [comment]
@MhLiao   Thank you very much for your careful reply and patient answers to every question. But this way we get a character box that only frames part of the character, not the whole character, right?

**Comment by MhLiao** [comment]
@HHeracles Right

**Comment by HHeracles** [comment]
@MhLiao Thanks for  your reply! And I want to ask another question. When fine-tuning training, we used multiple data sets,If we set batchsize to 1, can we train normally?Regardless of the time efficiency of training, whether the correct training results can be obtained
